app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, counter_cache: 'comments_counter'
+end
```

Comment:
- [ ] **Required:** You are required to have a method that updates the counter when a new comment is added. Kindly add the method.